### ISSUE TITLE
Hyperparameter Search

### ISSUE THREAD

**Issue by ssmmoo1** [ssmmoo1/RL_taskset_scheduling#11]
The model works in the single processor case and gets very close to EDF scores. Need to search learning rate, number of hidden features, number of output features, number of layers and MLP vs Dot Product predictor on a common test case.

**Comment by ssmmoo1** [comment]
done

This worked best from empirical testing. 

8 hidden features
4 output features
2 SAGE layers
MLP for link prediction
